docs/docs/sourcing-from-seams-cms.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Sourcing from Seams-CMS
+---
+
+## Overview
+
+[Seams-CMS](https://seams-cms.com) is a headless CMS system that allows you to easily manage content and data with either simple or complex structures and supports localization.
+
+Creating a Seams-CMS account is free for small hobby or small sites. For this tutorial, we will be using the demo workspace that is automatically created when you create a new account. Make sure you add an API key to this workspace. We will be needing your workspace ID and the (read-only) API key shortly.
+
+## Setup
+
+Create a new Gatsby site with the [default starter](https://github.com/gatsbyjs/gatsby-starter-default):
+
+Run this in your terminal:
+
+`gatsby new seamscms-site && cd seamscms-site`
+
+### Install the source plugin
+
+Next, we will be installing the Seams-CMS gatsby source plugin:
+
+`yarn add seams-cms-gatsby-source`
+
+### Adding configuration
+
+When you have installed the plugin, it's time to configure it. Add the following code to the `gatsby-config.js` file found in the root of your project.
+
+```javascript:title=gatsby-config.js
+module.exports = {
+  ...
+  plugins: [
+    ...
+    {
+      // Seams-CMS source plug
+      resolve: 'seams-cms-gatsby-source',
+      options: {
+        apiKey: <API_KEY>,
+        workspace: <WORKSPACE>,
+        contentTypes: ['blogpost'],
+      },
+    }
+    ...
+  ],
+  ...
+}
+```
+
+Note that `<API_KEY>` and `<WORKSPACE>` should come from your own Seams-CMS account. Inside the `contentTypes` array, we define which content types we want to download from Seams-CMS into GatsbyJS. This should also map to the names of your content type (API id's) that you will be sourcing.
```

Comment:
Is <WORKSPACE> same as "API workspace ID"? I think maybe add a bit more description, as it might be confused for the API key. Also, maybe state that the API key has to be generated.

Reply:
mhh:
> (API id's)

this is confusing - there is no `API id` mentioned before.

the ID should be consistent: `API ID` (uppercased like in `workspace ID`)
or is it the `API key` here?

Reply:
> <WORKSPACE>

if it the `workspace ID` then i would rename this to `<WORKSPACE_ID>`

Reply:
I've updated the text by clarifying things a bit and renaming to workspace_ID.  Hope this makes a bit more sense now..